src/main/resources/view/CommandBox.fxml

```diff
@@ -5,7 +5,7 @@
 <?import javafx.scene.control.TextField?>
 <?import javafx.scene.layout.StackPane?>
 
-<StackPane styleClass="stack-pane" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1">
+<StackPane styleClass="stack-pane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Shouldn't it be 11?

Reply:
Base AB3 Javafx version is 8. Version 11 should be referring to Java SDK.

Reply:
Oh i see.